go/vt/tabletserver/config.go

```diff
@@ -20,6 +21,9 @@ import (
 	topodatapb "github.com/youtube/vitess/go/vt/proto/topodata"
 )
 
+// longTime is for setting long timeouts, practically forever.
+const longTime = 24 * 365 * 10 * time.Hour
```

Comment:
I agree with Alain here. From the value, there's a subtle implication here that Vitess supports very long streaming queries, when there are numerous reasons to discourage day-long queries. I'd prefer using a reserved value that doesn't have an obvious time-scale meaning.